src/crypto/mod.rs

```diff
@@ -8,43 +8,8 @@ type ChecksumAlg = sha2::Sha512Trunc256;
 const CHECKSUM_LEN: usize = 4;
 const HASH_LEN: usize = 32;
 
-/// Public key address
```

Comment:
I like the idea of a crypto mod with an inner address mod, but I would keep Address and MultisigAddress together. The crypto mod will certainly contain more submodules soon (maybe ApiToken currently in utils?).

Reply:
I haven't put much thought into this yet, but you may want to separate the core node crypto (address, digest, multisig, etc) from the API-specific stuff.

So far, the option I like the most is to have a `core` crate with the low level node-related functionality (ie: block, tx, address, crypto, etc...), and put the API/SDK specific things in a separate crate (ie: endpoints, API tokens, etc).

Reply:
I haven't put much thought into this yet, but you may want to separate the core node crypto (address, digest, multisig, etc) from the API-specific stuff.

So far, the option I like the most is to have a core crate with the low level node-related functionality (ie: block, tx, address, crypto, etc...), and put the API/SDK specific things in a separate crate (ie: endpoints, API tokens, etc).

Reply:
Very good point, that's definetely the way to go!